### PR TITLE
SRC URI and branch change for hal-wifi-generic

### PR DESCRIPTION
**Pull request by manigandanj**
Signed-off-by: manigandan.gopalakrishnan <manigandan.gopalakrishnan@ltts.com>

### CODE REVIEW

File: recipes-ccsp/hal/hal-wifi-generic_git.bbappend [hunk 1]
```diff
@@ -1,17 +1,10 @@
-SRC_URI += "${CMF_GITHUB_ROOT}/rdkcentral/rdkb-turris-hal;protocol=${CMF_GIT_PROTOCOL};branch=${CMF_GIT_MASTER_BRANCH};destsuffix=git/source/wifi/devices;name=wifihal-turris"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```
Comment (scthunderbolt) [inline recipes-ccsp/hal/hal-wifi-generic_git.bbappend]: Why is this line needed? There does not seem to be patches in this change.

File: recipes-ccsp/hal/hal-wifi-generic_git.bbappend [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
 
-SRCREV = "${AUTOREV}"
+SRC_URI = "git://git@github.com/rdkcentral/rdkb-turris-hal.git;protocol=ssh;branch=turris-rdk-next;name=wifihal"
```
Comment (scthunderbolt) [inline recipes-ccsp/hal/hal-wifi-generic_git.bbappend]: The CMF variables are required for CMF releases and branching. Please use the variables instead of hard coded values. Also ssh is not supported.

**Comment by scthunderbolt** [comment]
The change should not be merged without updates to address the comments.

**Comment by manigandanj** [comment]
@scthunderbolt , 
Thanks for your involvement.
This PR is not required anymore .
Addressed our issue through another PR https://github.com/rdkcentral/rdkb-turris-hal/pull/34. Thus closing this PR.
